.buildkite/utils.jl

```diff
@@ -126,7 +126,7 @@ function build_step(NAME, PLATFORM, PROJECT)
         :commands => [script],
         :env => build_env,
         :artifacts => [
-            "**/products/$(first(split(NAME, "@")))*.tar.gz"
+            "**/products/$NAME*.tar.gz"
```

Comment:
Shouldn't be needed anymore after https://github.com/JuliaPackaging/Yggdrasil/commit/fadb9da33d0ef9a28c671ade05800849dabc17c7. And the whole reason this failed is because the version isn't part of the NAME, resulting in duplicate keys, so removing the version is a no-op.